src/person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative './nameable'
+require_relative './rental'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Same for rentals, it should be getter and setter just as the previous comment.